containers/Ideas/Idea.js

```diff
@@ -54,10 +75,13 @@ export const Idea = (props) => {
         setMenuVisible(false)
         props.navigation.navigate(
           'ReportCreateMessage',
-          {content_type: ideaState.content_type, object_pk: ideaState.pk}
+          {
+            content_type: ideaState.content_type,
+            object_pk: ideaState.pk
+          }
         )
       },
-      isFirst: !ideaState.has_changing_permission && !ideaState.has_deleting_permission,
+      isFirst: noPermissions,
```

Comment:
if we use the noPermissions variable only once, I like the old version more as it is easier to read - with the rules directly where they are used.

Reply:
do you mean not to use `noPermissions`? the reason is what @fuzzylogic2000 wrote, that it is easier to read inline and it is used only once

Reply:
sorry, I thought it still read the `noPermission` thing and was unresolved